src/ForkCMS/Bundle/InstallerBundle/Service/RequirementsChecker.php

```diff
@@ -278,13 +278,21 @@ protected function checkAvailableFiles()
     /**
      * Ensure that Apache .htaccess file is written and mod_rewrite does its job
      */
-    protected function checkApacheRewrites()
+    protected function checkServerRewrites()
     {
-        $this->checkRequirement(
-            'modRewrite',
-            (bool) (getenv('MOD_REWRITE') || getenv('REDIRECT_MOD_REWRITE')),
-            self::STATUS_WARNING
-        );
+        if (array_key_exists('SERVER_SOFTWARE', $_SERVER) && stripos($_SERVER['SERVER_SOFTWARE'], 'nginx') !== 0) {
+            $this->checkRequirement(
+                'modRewrite',
+                (bool) (getenv('MOD_REWRITE') || getenv('REDIRECT_MOD_REWRITE')),
+                self::STATUS_WARNING
+            );
+        } else {
```

Comment:
you don't need this else if you add a return in the if like in the code example I provided :)